package.json

```diff
@@ -27,7 +27,8 @@
   },
   "homepage": "https://github.com/NerdWalletOSS/apollo-cache-policies#readme",
   "peerDependencies": {
-    "@apollo/client": "^3.3.0"
+    "@apollo/client": "^3.3.0",
+    "react": "^16.0.0"
```

Comment:
Introduces react because of the new hooks

Reply:
Probably a semantic question, do you think it makes more sense to build this out into our `nwjs-apollo` lib in order to keep this independent of react?

Reply:
The hooks require new APIs I've added to the cache/client here so I think this lib makes the most sense for now. In the future as the functionality of this lib continues to expand we could look into modularizing it more potentially.

Reply:
Should this target v16.8 since [that's when hooks were added to React](https://reactjs.org/docs/hooks-intro.html)?

Reply:
Makes sense! Updated